main.c

```diff
@@ -561,6 +790,27 @@ int main(int argc,char **argv)
 				else
 					e_additional();
 			}
+			else if (*arg == 'O') {
+				if (argc--)
+					outfilekeys = *argv++;
```

Comment:
not happy about this way of signaling output mode.
should instead have something like `-Y` (yaml output) option and leave `-o` for specifying file.
this way one could have yaml output to stdout and use unix redirection instead of `tail -F`.
if file output is desired `-Y -o filename.yaml` could be used.

Reply:
actually maybe `-y` for "switch to yaml output" and `-Y` for "parse generated yaml output to directory"

Reply:
outputting to stdout by default makes it useful to for example pipe via netcat

Reply:
At first I really wanted to just print some plain text. But then I realize that with few changes in formatting this will become Yaml and this will preserve both human readability and machine parsability. I suppose that we can keep it as default format. Then could be added Json, XML, etc. But this seems not really useful as long as we already have strongly parsable format. 

Another point is the usage context of this tool. It's not about integration with everything. When somebody really need to do it, then probably he will run it for days or on a bunch of machines, and should be able to teach other-end system to parse Yaml, or make a simple trancoder between them.

> -Y for "parse generated yaml output to directory"

If you mean "put everything from file to FS" then I don't sure if this is good idea.
One of the main reasons of this patch is to minimizing FS pollution when I analyzing current results. And reason for minimizing is that my PC currently make 5900 keys a day for prefix of 5 symbols. After few weeks I'll pick something, but before I need to play with dictionaries, try some heuristics about memorability on all currently available candidates.
So usually there's never need to recreate all the content of the file on FS, but just one or few host need it.
And this way we can be ready for GPU future.

> outputting to stdout by default makes it useful to for example pipe via netcat

Totally agree :+1:

Reply:
no you misunderstood. what I'm saying, is that we should use same character, just lowercase `-y` for yaml output, and uppercase `-Y` for yaml decoding to filesystem.
I don't think we need any more than yaml, because it has well-defined streaming behavior compared to json and xml

Reply:
so that we can do something like `mkp224o -y beeep | mkp224o -Y`, or `mkp224o -y beeep | netcat 192.168.0.6 12345` + `netcat -l 12345 | mkp224o -Y` :)
using same character would make it easier to remember.

Reply:
`-O` -> `-y`
`-i` -> `-Y`